ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotSelectorFromSafeBlock.java

```diff
@@ -44,6 +44,8 @@ public class PivotSelectorFromSafeBlock implements PivotBlockSelector {
   private final Supplier<Optional<ForkchoiceEvent>> forkchoiceStateSupplier;
   private final Runnable cleanupAction;
 
+  private long lastNoFcuReceivedInfoLog = System.currentTimeMillis();
+  private static final long ONE_MIN_LOGGING_THRESHOLD = 60000L;
```

Comment:
```suggestion
  private static final long NO_FCU_RECEIVED_LOGGING_THRESHOLD = 60000L;
```

Reply:
nit: to match elsewhere we use similar naming eg SKIPPED_MESSAGES_LOGGING_THRESHOLD or ENGINE_API_LOGGING_THRESHOLD